src/main/scala/edu/berkeley/ce/rockslicing/CommandReader.scala

```diff
@@ -0,0 +1,54 @@
+package edu.berkeley.ce.rockslicing
+
+/** A simple tool for parsing command line inputs
+  * @constructor Create a new command reader
+  */
+object CommandReader {
+  val usage =
+    """
+       Usage: [spark arguments] sparkRocks-assembly-1.0.jar -inputFile [inputFile] -numberSeedJoints [Int] [output opts]
+         where:
+              [spark arguments]    See www.spark.apache.org for details on how to submit Spark applications
+              [inputFile]          File that contains input information on the rock volume of interest and joint sets
+              [Int]                Integer that specifies the number of joints to process before initiating parallel
+                                   instances
+         and [output opts] is one or all of the following
+              -toVTK               Generate output that can be converted to VTK format by rockProcessor
+              -toIE                Generate output that uses inequalities to represent rock blocks along with the
+                                   blocks' center of mass
+    """.stripMargin
+
+  type OptionMap = Map[Symbol, String]
+
+  def argumentParser(inputs: Array[String]): OptionMap = {
```

Comment:
It seems confusing to make the name of this method a noun, since it's more about taking an action than making a thing. How about `parseArguments` instead?